module/VuFind/tests/integration-tests/src/VuFindTest/View/Helper/Root/ResultFeedTest.php

```diff
@@ -57,7 +57,6 @@ public function setUp(): void
         // Give up if we're not running in CI:
         if (!$this->continuousIntegrationRunning()) {
             $this->markTestSkipped('Continuous integration not running.');
-            return;
```

Comment:
This change seems unrelated to the rest of the PR. I don't object to it, but just curious why it's here.

Reply:
Unless you really feel strongly that it's necessary, I think reverting makes sense, since it's out of scope for this PR and might have negative side effects in future.